samples/HelloWorld/.vs/HelloWorld/xs/UserPrefs.xml

```diff
@@ -0,0 +1,17 @@
+﻿<Properties StartupConfiguration="{1AB80089-B119-43EA-4271-3150731F769E}|">
```

Comment:
You don't need to commit anything under the samples directory. This PR has 284 files changed. You should figure out how to remove these from your diff. You of course don't want to just delete them from your repo, just make git stop tracking them.